src/ui/template.rs

```diff
@@ -0,0 +1,21 @@
+use super::{prompt_yes_no, Ui};
+use crate::{command, service, RP};
+use anyhow::Result;
+use std::io::Write;
+
+pub fn get_template(ui: &mut Ui<impl RP>, params: command::Template) -> Result<()> {
+    let template_parameters = service::create_template_parameters(&mut ui.res, &params)?;
+    let template_response = service::get_template(&mut ui.res, &template_parameters)?;
+    if service::file_exists(&ui.res, &template_response.filename) {
+        let overwrite_message = format!("A file ./{} already exists. Are you sure you want to overwrite the file with the downloaded template (yes/No)? ", &template_response.filename);
```

Comment:
Maybe split the message to two lines?

Reply:
@kluopaja Not just the literal, the printed line is also quite long.

Reply:
Is the new version ok?